### PR TITLE
subscriber: Forward current_span impl for Layered

### PR DESCRIPTION
**Pull request by LucioFranco**

### CODE REVIEW

File: tracing-subscriber/src/layer.rs [hunk 1]
```diff
@@ -501,6 +501,10 @@ where
         }
     }
 
+    fn current_span(&self) -> span::Current {
```
Comment (hawkw) [inline tracing-subscriber/src/layer.rs]: this should probably be `#[inline]`, like the other functions that forward to the inner subscriber?

Comment (LucioFranco) [reply tracing-subscriber/src/layer.rs]: done

**Comment by LucioFranco** [comment]
@hawkw squashed and added a message.

**Comment by hawkw** [comment]
> @hawkw squashed and added a message.

<3

**Pull request merged**